### PR TITLE
add integration tile for Rollbar

### PR DESCRIPTION
**Pull request by ezarowny**
@irabinovitch looking to get Rollbar added to Datadog's integration tiles

### CODE REVIEW

File: Rollbar/description.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    <div class="integration_content"  id="int-configuration">
+      <h2>Configuration</h2>
+      <p>
+        <em>Configuration is per-project in Rollbar.</em>
+      </p>
+      <ol>
+        <li>In Rollbar, go to the Notification settings page for a Project: <strong>Dashboard → Settings → Notifications → Datadog</strong></li>
+        <li>On Datadog, head to the APIs page to get your API Key (copy to your clipboard): <strong>Integrations → APIs</strong></li>
+        <li>Add your <strong>Datadog API key</strong> (API key here? How do we list the key?) in Rollbar</li>
```
Comment (ezarowny) [inline Rollbar/description.html]: @irabinovitch how do I actually get the user's API key in here?

File: Rollbar/description.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      <p>Connect Rollbar to Datadog to</p>
+      <ul>
+        <li>Get notified of exceptions, errors, code deployments in your event stream</li>
+        <li>Filter notifications by severity, environment, host, users and more</li>
+        <li>Search for exceptions in your graphs</li>
+        <li>Discuss exceptions with your team</li>
+        <li>Spend less time debugging issues</li>
+      </ul>
+      <p>
+        <img class="screenshot" src="${img_path(" rollbar-error.png ")}" />
```
Comment (ezarowny) [inline Rollbar/description.html:44]: @irabinovitch I think I did this right?

Comment (irabinovitch) [inline Rollbar/description.html:44]: looks good to me.

File: Rollbar/description.html (hunk 1 above)
Comment (irabinovitch) [inline Rollbar/description.html]: ${api_key} will be replaced with the users key at display time.

Comment (ezarowny) [inline Rollbar/description.html]: Ah cool, I will update that. Was that documented anywhere?

Comment (irabinovitch) [inline Rollbar/description.html]: It is not.  We're in the process of streamlining the tile process, including adding new docs.

**Comment by ezarowny** [comment]
@irabinovitch what's the process for updating the tile later on?

**Comment by ezarowny** [comment]
@irabinovitch also, anything you need me to change?

**Comment by themlsmith** [comment]
@irabinovitch Hi. When do you think this will be merged and deployed?

**Comment by irabinovitch** [comment]
@ezarowny  @themlsmith This is out in production.

**Comment by ezarowny** [comment]
Thanks @irabinovitch!

**Pull request closed without merging**